### ISSUE TITLE
typo in engines.node 

### ISSUE THREAD

**Issue by perrefe** [medikoo/memoizee#136]
There is a typo in engine version specification in package.json for v0.4.16 release

```
"engines": {
    "node": ">=.0.12"
},
```

```
error memoizee@0.4.16: The engine "node" is incompatible with this module. Expected version ">=.0.12". Got "20.11.1"
error Found incompatible module.
```

I'm using yarn v1.22.19 and node 20.11.1

I think it should be `"node": ">=12.0"`

thanks!

**Comment by NicoHinderling** [comment]
+1 im also facing this right now

**Comment by perrefe** [comment]
Using npm 10.5.2 shows this warning

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'memoizee@0.4.16',
npm WARN EBADENGINE   required: { node: '>=.0.12' },
npm WARN EBADENGINE   current: { node: 'v20.13.1', npm: '10.5.2' }
npm WARN EBADENGINE }
```

**Comment by pcollaog** [comment]
+1

**Comment by Blari** [comment]
As a temporary solution until the author fixes the typo, you can add the following to your package.json file:

For yarn install:

```json
"resolutions": {
  "memoizee": "0.4.14"
}
```
For npm install:
```json
"overrides": {
  "memoizee": "0.4.14"
}
```

**Comment by tsuyoda** [comment]
+ 1

**Comment by simlu** [comment]
https://github.com/medikoo/memoizee/commit/1a6989ff797e415625db405f3e52094f72aaa5a4

**Comment by diegodleon** [comment]
+1

**Comment by galaxyreal** [comment]
Thanks for @Blari 's solution.
Additionally, Memoizee 0.4.15 is working as well.
```
"resolutions": {
  "memoizee": "0.4.15"
}
```

**Comment by lynnzc** [comment]
> As a temporary solution until the author fixes the typo, you can add the following to your package.json file:
> 
> For yarn install:
> 
> ```json
> "resolutions": {
>   "memoizee": "0.4.14"
> }
> ```
> 
> For npm install:
> 
> ```json
> "overrides": {
>   "memoizee": "0.4.14"
> }
> ```

this works, 👍

**Comment by frederichoule** [comment]
+1

**Comment by DaniloMarques1** [comment]
Also facing the problem

**Comment by simlu** [comment]
Duplicate fyi https://github.com/medikoo/memoizee/issues/135

**Comment by ACC-Ricky** [comment]
+1

**Comment by 97thjingba** [comment]
+1

**Comment by danharten-sovtech** [comment]
+1

**Comment by thoroc** [comment]
PR in https://github.com/medikoo/memoizee/pull/137

**Comment by 97thjingba** [comment]
> PR in #137

It seems that it will take some time to merge and deploy

**Comment by medikoo** [comment]
Duplicate of #135

**Comment by medikoo** [comment]
(also fixed and published with v0.4.17)
